env/src/test/java/com/sap/cloud/security/config/EnvironmentsK8sTest.java

```diff
@@ -5,23 +5,40 @@
  */
 package com.sap.cloud.security.config;
 
+import com.sap.cloud.environment.servicebinding.api.DefaultServiceBindingAccessor;
+import com.sap.cloud.environment.servicebinding.SapServiceOperatorLayeredServiceBindingAccessor;
 import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
 import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
 
+import java.nio.file.Paths;
+
+import static com.sap.cloud.environment.servicebinding.SapServiceOperatorLayeredServiceBindingAccessor.DEFAULT_PARSING_STRATEGIES;
 import static com.sap.cloud.security.config.k8s.K8sConstants.KUBERNETES_SERVICE_HOST;
 
 @ExtendWith(SystemStubsExtension.class)
 class EnvironmentsK8sTest {
 
 	private static final String K8S_HOST_VALUE = "0.0.0.0";
 
-	@Test
-	void getCurrent_returnsOnlySingleK8sInstance(EnvironmentVariables environmentVariables) {
+	@BeforeAll
+	static void beforeAll(EnvironmentVariables environmentVariables) {
 		environmentVariables.set(KUBERNETES_SERVICE_HOST, K8S_HOST_VALUE);
+		DefaultServiceBindingAccessor.setInstance(new SapServiceOperatorLayeredServiceBindingAccessor(
+				Paths.get("src/test/resources/k8s"), DEFAULT_PARSING_STRATEGIES));
```

Comment:
Hardcoding the resource path like this is prone to error IMO.
AFAIK, the best practice is to get the `ClassLoader` of this test class and then get a resource from there.

Look at [this implementation](https://github.com/SAP/btp-environment-variable-access/blob/main/sap-service-operator/src/test/java/com/sap/cloud/environment/servicebinding/TestResource.java#L16-L57) (and [the usage](https://github.com/SAP/btp-environment-variable-access/blob/main/sap-service-operator/src/test/java/com/sap/cloud/environment/servicebinding/SapServiceOperatorLayeredServiceBindingAccessorTest.java#L29)) for an example.

Reply:
Done with `Paths.get(K8sEnvironmentTest.class.getResource("/k8s").getPath())`